controllers/remote/cluster.go

```diff
@@ -50,7 +50,7 @@ func NewClusterClient(ctx context.Context, sourceName string, c client.Client, c
 
 // RESTConfig returns a configuration instance to be used with a Kubernetes client.
 func RESTConfig(ctx context.Context, sourceName string, c client.Reader, cluster client.ObjectKey) (*restclient.Config, error) {
-	kubeConfig, err := kcfg.FromSecret(ctx, c, cluster)
+	kubeConfig, err := kcfg.FromSecret(ctx, c, cluster, true)
```

Comment:
I think this should probably be `false` so that the behaviour stays the same (i.e. get the kubeconfig used by capi, a.k.a system kubeconfig)